cinn/frontend/op_mappers/fetch_feed.cc

```diff
@@ -0,0 +1,30 @@
+#include "cinn/common/macros.h"
+#include "cinn/frontend/op_mapper_registry.h"
+#include "cinn/frontend/op_mappers/common_utils.h"
+
+namespace cinn {
+namespace frontend {
+namespace op_mappers {
+
+void FetchOpMapper(const paddle::cpp::OpDesc& op_desc, const OpMapperContext& ctx) {
+  CHECK_EQ(op_desc.Input("X").size(), 1UL);
+  auto output_name = op_desc.Input("X").front();
+  LOG(INFO) << "detect model output: [" << output_name << "]";
+}
+
+void FeedOpMapper(const paddle::cpp::OpDesc& op_desc, const OpMapperContext& ctx) {
+  auto outs = op_desc.Output("Out");
+  CHECK_EQ(outs.size(), 1UL);
+  VLOG(2) << "Model get feed [" << outs[0] << "]";
+  Placeholder input(common::Float(32), {}, outs[0]);
```

Comment:
建议这里使用builder.CreateInput接口创建输入。
```suggestion
auto input = ctx.builder_->CreateInput(common::Float(32), {}, outs[0]);
```

Reply:
好的，我提个PR修复下